.github/workflows/brew.yml

```diff
@@ -0,0 +1,19 @@
+name: brew
+
+on:
+  release:
+    types: published
+
+jobs:
+  homebrew:
+    name: Bump homebrew-core formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v2
```

Comment:
You should use https://github.com/dawidd6/action-homebrew-bump-formula since `aws-sam-cli` has Python resources that need updating, which `mislav/bump-homebrew-formula-action` does not update.